test/performance/latency_test.go

```diff
@@ -31,17 +32,26 @@ import (
 	"github.com/knative/test-infra/shared/testgrid"
 )
 
-func TestTimeToServeLatency(t *testing.T) {
+const (
+	sleepTime = 1 * time.Minute
+	// sleepReqTimeout should be > sleepTime. Else, the request will time out before receiving the response
+	sleepReqTimeout = 2 * time.Minute
```

Comment:
nit: I would prefer naming it as `reqTimeout` as the `sleep` prefix is a bit confusing.

Reply:
there are 2 const here for sleep and helloworld images. This referes to the timeout image reqTimeout, while the other refers to hellowworldreqTimeout.